nvd3/templates/content.html

```diff
@@ -115,6 +115,11 @@
         {{ chart.extras }}
     {% endif %}
 
+    {# callback for clicking on charts #}
+    {% if chart.callback %}
+       },{{ chart.callback }});
```

Comment:
This is such that the passed in string includes only the javascript function (looks more intuitive) rather than closing the previous structure.  An example will explain better.  I will add that.